### PR TITLE
Disposable/track vessel

### PR DESCRIPTION
**Pull request by apercas**

### CODE REVIEW

**Comment by tiagojsag** [comment]
nice branch name ;) the change set seems quite small, can you make sure that everything's in here, please? Also, add @davidsingal as CRer plesase

**Comment by apercas** [comment]
Yes, everything's there, what I needed was just a couple of nested loops plus the `polyline` Google Maps function :)

File: app/src/components/map.jsx [hunk 1]
```diff
@@ -160,11 +164,14 @@ class Map extends Component {
 						    />
 						  }
         googleMapElement={
-								<GoogleMap ref="map" defaultZoom = {0} defaultCenter = {{lat: 0, lng: 0}} defaultMapTypeId = {google.maps.MapTypeId.SATELLITE}
+								<GoogleMap ref="map" defaultZoom = {3} defaultCenter = {{lat: 0, lng: 0}} defaultMapTypeId = {google.maps.MapTypeId.SATELLITE}
```
Comment (davidsingal) [inline app/src/components/map.jsx]: I think is better put this props in a default settings object or something like that.

**Comment by davidsingal** [comment]
We can make a squash in this branch.

**Comment by apercas** [comment]
I think it'd be better to do that when merging into master, just in case.

**Comment by davidsingal** [comment]
Maybe... But first commit and second commit touch the same file and it's harder to review.

**Comment by tiagojsag** [comment]
@apercas squash/rebase should never happen on shared branches - so if you squash when merging from develop to master, you are leaving github's develop branch inconsistent with everyone's local branches, which will lead to conflict nightmare. please squash these commits here

**Pull request merged**